Components/avatar/avatar.css

```diff
@@ -0,0 +1,36 @@
+.avatar{
+    --size:50px;
+    height: var(--size);
+    width: var(--size);
+    border-radius: 100%;
+
+    color: white;
+    background:grey;
```

Comment:
suggestion use variables here

Reply:
I will use variables eventually. Right now I'm not sure about the colors so I thought to go with basic colors and when the component library becomes complete replace all with variables.